modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/spatialindexderivation/v0.1.0.json

```diff
@@ -0,0 +1,44 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
+    "{{base}}/contexts/bbp/core/entity/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "this": "{{base}}/schemas/hbphackaton/simulation/spatialindexderivation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/nexus/prov/derivation/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:SpatialIndexDerivationShape",
+      "@type": "sh:NodeShape",
+      "label": "Spatial index derivation shape definition",
+      "targetClass": "prov:SpatialIndexDerivation",
```

Comment:
nsg:SpatialIndexDerivation

Reply:
It's preferable to use nsg:SpatialIndexDerivation as the users will need to differentiate it with other prov:Derivation